### ISSUE TITLE
Missing UI feedback, checkout action clickable multiple times

### ISSUE THREAD

**Issue by hediet** [microsoft/vscode#186321]
Testing #186170

When clicking on "Check out" you can actually keep clicking, as the hover does not disappear immediately and there is no direct UI feedback:

![Code_-_Insiders_Cm2kR95kK1](https://github.com/microsoft/vscode/assets/2931520/c40f459e-3534-4822-a1f3-8f966cc047dc)

I suggest to hide the hover immediately on click and to make sure the notification that checking out is in progress shows quicker (without delay).

**Comment by alexr00** [comment]
I agree that we should show an indication of progress faster, but I don't think we should hide the hover on click.
